lib/config_builder/version.rb

```diff
@@ -1,3 +1,3 @@
 module ConfigBuilder
-  VERSION = '0.12.0'
+  VERSION = '0.12.1'
```

Comment:
Please bump this in a separate commit, I prefer to do release commits that bump the version and update the changelog in the same commit.